bibliopixel/drivers/APA102.py

```diff
@@ -5,10 +5,8 @@
 class APA102(DriverSPIBase):
     """Main driver for APA102 based LED strips on devices like the Raspberry Pi and BeagleBone"""
 
-    def __init__(self, num, c_order=ChannelOrder.RGB, use_py_spi=True,
-                 dev="/dev/spidev0.0", SPISpeed=2, open=open):
-        super().__init__(num, c_order=c_order, use_py_spi=use_py_spi, dev=dev,
-                         SPISpeed=SPISpeed, open=open)
+    def __init__(self, num, c_order=ChannelOrder.RGB, **kwargs):
+        super().__init__(num, c_order=c_order, **kwargs)
```

Comment:
I'm with you to have a common code style.

So should I full classify the arguments without the `**kwargs` or only document  that in the Manifest?